sql/schema.sql

```diff
@@ -6,31 +6,31 @@
 -----------------
 
 CREATE TABLE IF NOT EXISTS repos
-( id               INT  NOT NULL
-, repo_name        TEXT NOT NULL
-, repo_desc        TEXT NOT NULL
-, repo_url         TEXT NOT NULL -- The repo html_url
-, open_issue_count INT  NOT NULL
-, fork_count       INT  NOT NULL
+( id         SERIAL       
+, repo_name  TEXT   NOT NULL
+, repo_owner TEXT   NOT NULL
+, repo_desc  TEXT
```

Comment:
Let's not prefix columns with the `repo_` or table name. It is a little bit redundant because during SQL queries we can always refer to columns by fully qualified name like `repos.name` or `repos.owner` in case of ambiguity. So in tables, in general, it's better to have ordinary names.

Reply:
Got it.